app/models/service_template_ansible_playbook.rb

```diff
@@ -193,10 +193,8 @@ def create_dialogs(config_info)
   end
 
   def delete_job_templates(job_templates)
```

Comment:
Is this the spec [here](https://github.com/jameswnl/manageiq/blob/f4759d2f4958378896aa20b2e073a54b5140dd8b/spec/models/service_template_ansible_playbook_spec.rb#L255-L255) ?

Can you send me the stack trace? Let me try.

I am not very familiar with the service template flow.  Would be much easier if there's a spec that can cover your case.